patches/chromium/fix_crash_when_saving_edited_pdf_files.patch

```diff
@@ -13,18 +13,476 @@ This patch can be removed should we choose to support chrome.fileSystem
 or support it enough to fix the crash.
 
 diff --git a/chrome/browser/resources/pdf/pdf_viewer.ts b/chrome/browser/resources/pdf/pdf_viewer.ts
-index 2ce9eaccfe7d9f39a14f088fdfdf8fd4b0ef85ab..f0136a8da926471798588670d3dd062f081bcb7a 100644
+index 63e6e4ea24cd7e1df3963096b54573b4acfccedf..26caec6fbc95d4ba0f9d18b9ae141cd27566d757 100644
 --- a/chrome/browser/resources/pdf/pdf_viewer.ts
 +++ b/chrome/browser/resources/pdf/pdf_viewer.ts
-@@ -900,26 +900,12 @@ export class PdfViewerElement extends PdfViewerBaseElement {
+@@ -15,32 +15,32 @@ import './pdf_viewer_shared_style.css.js';
+ import 'chrome://resources/cr_elements/cr_hidden_style.css.js';
+ import 'chrome://resources/cr_elements/cr_shared_vars.css.js';
+ 
+-import {assert, assertNotReached} from 'chrome://resources/js/assert.js';
+-import {loadTimeData} from 'chrome://resources/js/load_time_data.js';
+-import {listenOnce} from 'chrome://resources/js/util.js';
+-
+-import {Bookmark} from './bookmark_type.js';
+-import {BrowserApi} from './browser_api.js';
+-import {Attachment, DocumentMetadata, ExtendedKeyEvent, FittingType, Point, SaveRequestType} from './constants.js';
+-import {MessageData, PluginController} from './controller.js';
++import { assert, assertNotReached } from 'chrome://resources/js/assert.js';
++import { loadTimeData } from 'chrome://resources/js/load_time_data.js';
++import { listenOnce } from 'chrome://resources/js/util.js';
++
++import { Bookmark } from './bookmark_type.js';
++import { BrowserApi } from './browser_api.js';
++import { Attachment, DocumentMetadata, ExtendedKeyEvent, FittingType, Point, SaveRequestType } from './constants.js';
++import { MessageData, PluginController } from './controller.js';
+ // <if expr="enable_ink">
+-import {ContentController} from './controller.js';
++import { ContentController } from './controller.js';
+ // </if>
+-import {ChangePageAndXyDetail, ChangePageDetail, ChangePageOrigin, NavigateDetail} from './elements/viewer-bookmark.js';
+-import {ViewerErrorDialogElement} from './elements/viewer-error-dialog.js';
+-import {ViewerPasswordDialogElement} from './elements/viewer-password-dialog.js';
+-import {ViewerPdfSidenavElement} from './elements/viewer-pdf-sidenav.js';
+-import {ViewerToolbarElement} from './elements/viewer-toolbar.js';
++import { ChangePageAndXyDetail, ChangePageDetail, ChangePageOrigin, NavigateDetail } from './elements/viewer-bookmark.js';
++import { ViewerErrorDialogElement } from './elements/viewer-error-dialog.js';
++import { ViewerPasswordDialogElement } from './elements/viewer-password-dialog.js';
++import { ViewerPdfSidenavElement } from './elements/viewer-pdf-sidenav.js';
++import { ViewerToolbarElement } from './elements/viewer-toolbar.js';
+ // <if expr="enable_ink">
+-import {InkController, InkControllerEventType} from './ink_controller.js';
++import { InkController, InkControllerEventType } from './ink_controller.js';
+ //</if>
+-import {LocalStorageProxyImpl} from './local_storage_proxy.js';
```

Comment:
@deepak1556 why all these formatting changes?  Seems unnecessary.

Reply:
Oops, formatting changes were not intended, seems to be an issue with editor configuration. The change was mostly about avoiding deleting line from upstream in favor of using an ` // <if expr="">` block, let me re-export the patch.